### PR TITLE
[NDS-505] group global theme properties and change usages

### PR DESCRIPTION
**Pull request by kostasdano**
## Description

[NDS-505](https://orfium.atlassian.net/browse/NDS-505) 

[part of [NDS-501](https://orfium.atlassian.net/browse/NDS-501)]

### CODE REVIEW

File: src/theme/index.ts [hunk 1]
```diff
@@ -19,17 +19,19 @@ const defaultTheme = (theming: ColorScheme): Theme => {
       : enhancePaletteWithShades(darkPaletteConfig);
 
   return {
-    palette,
-    typography,
-    spacing,
-    elevation,
+    globals: {
+      colors: palette,
```
Comment (panvourtsis) [inline src/theme/index.ts:23]: colors will not be a different one for each `theming`

so `colors: enhancePaletteWithShades(lightPaletteConfig)` where `enhancePaletteWithShades(lightPaletteConfig)` could be `enhancePaletteWithShades(paletteConfig)`

File: src/components/ThemeProvider/__snapshots__/ThemeProvider.stories.storyshot [hunk 2]
```diff
@@ -12,7 +12,7 @@ exports[`Storyshots Design System/ThemeProvider Setting up a button example 1`]
   font-size: 1rem;
   font-weight: 500;
   color: white;
-  background-color: #2d2d46;
+  background-color: #175BF5;
```
Comment (panvourtsis) [inline src/components/ThemeProvider/__snapshots__/ThemeProvider.stories.storyshot:15]: why this changed now?

File: src/theme/index.ts [hunk 3]
```diff
@@ -5,31 +5,30 @@ import elevation from './elevation';
...
-      : enhancePaletteWithShades(darkPaletteConfig);
+  const palette = enhancePaletteWithShades(paletteConfig);
 
   return {
-    palette,
-    typography,
-    spacing,
-    elevation,
+    globals: {
+      colors: enhancePaletteWithShades(paletteConfig),
```
Comment (panvourtsis) [inline src/theme/index.ts:20]: why not using `palette` here?

Comment (kostasdano) [reply src/theme/index.ts]: oops, you are right 🐒

**Review by panvourtsis — APPROVED** [review]

**Pull request merged**